pie-dao/pools.piedao.org
# Feature/pie-548: eDOUGH Vesting UI

##### Description


##### Checklist


- [ ] Test coverage hasn't decreased

##### Testing
1. Check vesting entries = tot amount of positions at the table below
2. Check Vest in Dough = tot Vest in Dough positions with 'Can Claim' status
3. Check Vest in veDough = tot Vest in veDough positions with 'Can Stake' status
4. Check Claim
--> A modal should open when claiming, clocking on Claim will start the transacion and Claim button text has to change to 'Claiming...'
--> User should be prompted to confirm the transaction
--> Once transaction is complete, 'Claiming...' button text should change to 'claimed' and user redirected to /#/farms
--> Dough in the wallet
--> All Entries with status Can Claim will be removed from the list
5. Check Staking
--> Claiming should trigger the transaction without a modal, Stake button text to change to 'Staking...'
--> User should be prompted to confirm the transaction
--> Once transaction is complete, a modal will open to confirm the transaction
--> User to be redirected to /#/dough-staking
--> ve Dough in the wallet
--> /#/Dough-staking Summary widget to show show the added veDough
--> /#/Dough-staking Your Staking widget to show the staked Dough position
--> All Entries with status Can Stake will be removed from the list

##### Refers/Fixes
